### PR TITLE
Some minor cult fixes

### PR DESCRIPTION
**Pull request by Heheniy**
## About The Pull Request
look at title and code

## Why It's Good For The Game
make a guess

### CODE REVIEW

File: code/modules/antagonists/roguetown/villain/zizocult.dm [hunk 1]
```diff
@@ -169,7 +169,7 @@ GLOBAL_LIST_EMPTY(ritualslist)
 	set name = "Praise the Godhead!"
 	set category = "ZIZO"
 
-	if(stat == DEAD)
+	if(src.stat == DEAD)
```
Comment (Gyrandola) [inline code/modules/antagonists/roguetown/villain/zizocult.dm:172]: src seems redundant here, have you tested it?

File: code/modules/antagonists/roguetown/villain/zizocult.dm [hunk 2]
```diff
@@ -186,7 +186,7 @@ GLOBAL_LIST_EMPTY(ritualslist)
 	set name = "Communicate"
 	set category = "ZIZO"
 
-	if(stat == DEAD)
+	if(src.stat == DEAD)
```
Comment (Gyrandola) [inline code/modules/antagonists/roguetown/villain/zizocult.dm:189]: same as above

Comment (Heheniy) [reply code/modules/antagonists/roguetown/villain/zizocult.dm:189]: it is simply not working, game simply does not know whose dead stat it needs to check

**Comment by Gyrandola** [comment]
Show me test evidence for the stat thing and I'll approve it.

**Comment by Heheniy** [comment]
image
image

**Comment by Gyrandola** [comment]
> image image


image

image

**Comment by Heheniy** [comment]
give me a minute

**Comment by Heheniy** [comment]
you are right, i was confused with peole being able to praise zizo while being _almost_ dead

**Review by Gyrandola — APPROVED** [review]

**Pull request merged**